src/components/Article-comments.jsx

```diff
@@ -0,0 +1,81 @@
+import { React, useEffect } from "react";
+import { deleteComment } from "../utils/api";
+import { getComments } from "../utils/api";
+
+const ArticleComments = ({
+  open,
+  setComments,
+  comments,
+  commentChange,
+  setErr,
+  err,
+  setCommentChange,
+  username,
+  article_id,
+}) => {
+  useEffect(() => {
+    setCommentChange(false);
+    getComments(article_id)
+      .then((data) => {
+        function compare(a, b) {
```

Comment:
Extract this helper function and put it in a separate js file please :) It's always best practice to have a dedicated js file for helper functions.